bin/fabric-service-get-block.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ "$(uname)" = "Darwin" ]]; then
+    PROFILE_PATH=$(greadlink -f "$1")
+else
+    PROFILE_PATH=$(readlink -f "$1")
+fi
+
+
+. ${PROFILE_PATH}/settings
+
+echo "Getting system block by number $3 at channel $2..."
+
+SERVICE_URL="localhost:${SERVICE_BIND_PORT}"
+curl -k -G --silent --show-error \
+--key ${PROFILE_PATH}/crypto/users/admin/admin.key \
+--cert ${PROFILE_PATH}/crypto/users/admin/admin.crt \
+--request GET \
+--output "$4" \
+https://${SERVICE_URL}/admin/get-user-key \
```

Comment:
doesn't seems like it will give you a block...
and also doesn't seems to be related to this PR